example-next/pages/profile/[...members].js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable require-jsdoc */
+/* eslint-disable max-len */
+/*
+ TODO:
+   - [ ] determine dev/prod "paths" configuration for getStaticPaths (env var to set top z-index percentile on prod?)
+
+*/
+import axios from "axios";
+import {Title} from "@mantine/core";
+import {useTranslation} from "next-i18next";
+import {useRouter} from "next/router";
+import {serverSideTranslations} from "next-i18next/serverSideTranslations";
+import {Profile, NonIdealState} from "@datawheel/canon-next";
+import getProfile from "../../helpers/getProfile";
+// import {profileSearchConfig} from "../../helpers/search";
+const {NEXT_PUBLIC_CMS} = process.env;
+
+function ProfilePage({profile, formatters}) {
+  const router = useRouter();
+  const {t} = useTranslation("profile");
+
+  if (router.isFallback) return <NonIdealState />;
+  return (
+    <>
+      <Title>Profile</Title>
+      <Profile
+        formatters={formatters}
+        profile={profile}
+        linkify={profile => profile.reduce((href, member) => `${href}/${member.slug}/${member.memberSlug || member.id}`, "/profile")}
+        // you can also specify the configuration for ProfileSearch here:
+        // searchProps={{placeholder: "Seach profiles"}}
+        t={t}
+      />
+    </>
+  );
+}
+
+export default ProfilePage;
+
+
+export async function getStaticPaths() {
+  return {
+    paths: [
+      // Place here the paths you want to cache on build
+    ],
+    fallback: true
+  };
+}
+
+export async function getStaticProps({locale, params}) {
+  const profile = await getProfile(params.members, locale);
+  if (profile.error) return {notFound: true};
+  const formatterURL = new URL("/api/formatters", NEXT_PUBLIC_CMS);
+  const formatters = await axios.get(formatterURL.href)
+    .then(resp => resp.data);
+
+  return {
+    props: {
+      ...await serverSideTranslations(locale, ["common", "profile"]),
+      formatters,
+      profile
+    }
+  };
```

Comment:
Similar comment from `getStaticPaths`, can we move `getProfile` and all of this logic into:

```
return profileStaticPropsDefaults(); // I'm open to suggestions about a better/shorter function name
```